python/ray/experimental/channel/common.py

```diff
@@ -76,16 +76,16 @@ def set_contains_type(self, typ: "ChannelOutputType") -> None:
     def create_channel(
         self,
         writer: Optional["ray.actor.ActorHandle"],
-        readers: List[Optional["ray.actor.ActorHandle"]],
+        reader_to_node: List[Tuple["ray.actor.ActorHandle", str]],
```

Comment:
```suggestion
        readers_to_nodes: List[Tuple["ray.actor.ActorHandle", str]],
```

Reply:
How about changing all `reader_to_node` to `reader_to_node_id` instead?